lib/realtime/helpers.ex

```diff
@@ -87,7 +92,8 @@ defmodule Realtime.Helpers do
         pool \\ 5,
         queue_target \\ 5_000,
         ssl_enforced \\ true,
-        application_name \\ "realtime_supabase"
+        application_name \\ "realtime_supabase",
+        backoff_type \\ :exp
```

Comment:
Default for db connection is rand_exp right? Should we stick with that for the default?

Reply:
will change